index.d.ts

```diff
@@ -49,6 +49,18 @@ declare global {
     arrayBuffer(): Promise<ArrayBuffer>
     text(): Promise<string>
   }
+
+  type FigmaVirtualNode<T> = { __type: T }
+
+  type FigmaDeclarativeChildren<T> =
+    | FigmaVirtualNode<T>
```

Comment:
Where is `FigmaVirtualNode` coming from? Or is this what the widget typings are for?